operator/cmd/operator/server.go

```diff
@@ -134,6 +140,15 @@ func run() {
 		log.Fatalf("Could not create a controller manager: %v", err)
 	}
 
+	log.Info("Creating operator metrics exporter")
+	exporter, err := ocprom.NewExporter(ocprom.Options{
+		Registry: ctrlmetrics.Registry.(*prometheus.Registry),
```

Comment:
is the `ctrlmetrics.Registry` special in a way that we need here, or could we just use the `prometheus.DefaultRegisterer` ?

Reply:
K8s-controller-runtime already provides useful metrics like reconcile time histogram, counts etc. We provide metrics server settings while creating controller (like host, port, path) and internally k8s-controller-runtime publishes metrics to `ctrlmetrics.Registry`. If we publish our metrics to the same registry, then we can serve custom metrics in the same path, that is `/metrics`. 

Not sure if we can serve the metrics from two registries under the same endpoint. If that is the case then we can go with `prometheus.DefaultRegisterer`. Even if aggregating registries is possible, why do that?